README.md

```diff
@@ -52,7 +52,7 @@ Community discussions happen primarily on the [dev mailing list][dev-list] or on
 
 ### Building
 
-Iceberg is built using Gradle 5.4.1.
+Iceberg is built using Gradle 5.4.1 with *exact* Java 1.8.
```

Comment:
Isn't it enough to state that Iceberg is built with 1.8? Why add "exact" in bold?

Reply:
Make sense for marketing and we're working for Java 11. Updated